easybuild/easyconfigs/c/cuDNN/cuDNN-7.6.4.38-gcccuda-2019a.eb

```diff
@@ -0,0 +1,32 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# OriginalAuthor:    Stephane Thiell <sthiell@stanford.edu>
+# Author:    Ake Sandgren <ake.sandgren@hpc2n.umu.se>
+##
+easyblock = 'Tarball'
+
+# The full version of the library can be found using
+# strings -a cuda/lib64/libcudnn_static.a | grep cudnn_version_
+# Download and rename.
+name = 'cuDNN'
+version = '7.6.4.38'
+
+homepage = 'https://developer.nvidia.com/cudnn'
+description = """The NVIDIA CUDA Deep Neural Network library (cuDNN) is
+a GPU-accelerated library of primitives for deep neural networks."""
+
+# gcccuda 2019a uses CUDA 10.1
+toolchain = {'name': 'gcccuda', 'version': '2019a'}
```

Comment:
@akesandgren Why not use `system` toolchain like we do with `cuDNN-7.6.2.24-CUDA-10.1.243.eb`?